tests/acceptance/steps/util.py

```diff
@@ -66,20 +84,31 @@ def delete_topic(topic_name):
 
 def call_cmd(cmd):
     output = ''
+
+    communicate_input = b'y' if six.PY3 else 'y'
```

Comment:
`b'y'` should be both py2 and py3 compatible: http://python-future.org/compatible_idioms.html#byte-string-literals